### DISCUSSION TITLE
Change default branch name to main

### DISCUSSION THREAD

**Discussion by peterl3g** [community/community#63387] [New to GitHub] [answered]
Hey guys,
I have a quick question: My default branch name is master, but i saw that a lot of people are calling their default branch main, which i would prefer. How can i change it to be main out of the box ?

**Comment by w8ste** [comment, marked as answer]
@peterl3g Hi, here is what you need to do: 

### Local
- run in your terminal/cli-tool the following command:
```
git config --global init.defaultBranch main
```
### GitHub
- click on the profile icon in the top right corner 
- go to settings 
- under "Code, planning, and automation" you will "Repositories"
- There you can set the "Repository default branch" to main

I hope this helps :)
